### PR TITLE
Add Footer component

### PR DESCRIPTION
**Pull request by amberstarlight**
Adds a footer component that is rendered on every page. 🦶

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -19,6 +24,11 @@ function App() {
         <Route path="/admin" component={Admin} />
         <Route path="/" component={Home} />
       </Switch>
+
+      <Footer
+        repo="https://github.com/amberstarlight/cutieclubsite"
```
Comment (mini-ninja-64) [inline frontend/src/App.js]: Fetch this from package.json

File: frontend/src/components/Footer/Footer.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+function Footer(props) {
+  const phrase = randomItem(props.json.phrases);
+
+  return (
+    <footer>
+      <div className="container">
+        <img src={roundel} alt="Cutie Club" />
+        <div className="wrapper">
+          <div className="left">
+            <h2>{phrase}</h2>
```
Comment (mini-ninja-64) [inline frontend/src/components/Footer/Footer.js]:
```suggestion
            <h2>{randomItem(props.json.phrases)}</h2>
```
could do this not sure whats prefferential

File: frontend/src/App.js (hunk 1 above)
Comment (amberstarlight) [reply frontend/src/App.js]: We can't get this from `package.json` as we can't import outside of `src`.

**Comment by amberstarlight** [comment]
I've made these changes now. :+1:

**Pull request merged**